frontend/vite.config.ts

```diff
@@ -4,5 +4,13 @@ import tsconfigPaths from "vite-tsconfig-paths"
 
 // https://vitejs.dev/config/
 export default defineConfig({
-  plugins: [react(), tsconfigPaths()]
+  plugins: [react(), tsconfigPaths()],
+  server: {
+    watch: {
+      usePolling: true
+    },
+    host: true,
+    strictPort: true,
+    port: 5005
+  }
```

Comment:
do we really need this?

Reply:
ใช่ ตอนแรกไม่มี พอเข้า web ละมันหาไม่เจอ

Reply:
แป่ก ๆ แต่ช่างมัน